src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,10 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private final UserService newUserSerice = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User user1 = newUserSerice.findByEmail(email);
+        return user1 != null && password.equals(user1.getPassword());
```

Comment:
```suggestion
        User user = newUserSerice.findByEmail(email);
        return user != null && password.equals(user.getPassword());
```
actually it's a bad practice to add such suffixes to the var names